src/transport/manager/handle.rs

```diff
@@ -285,7 +287,7 @@ pub struct TransportHandle {
     pub protocols: HashMap<ProtocolName, ProtocolContext>,
     pub next_connection_id: Arc<AtomicUsize>,
     pub next_substream_id: Arc<AtomicUsize>,
-    pub protocol_names: Vec<ProtocolName>,
+    pub _protocol_names: Vec<ProtocolName>,
```

Comment:
dq: Do we plan to use this in the future?

Reply:
I think we can safely remove it for now (done).